cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -60,7 +56,7 @@ This section comprises a glossary of all the keywords -- grouped by category and
 |<<query-return, RETURN ... [AS]>>                    | Projecting   |  Defines what to include in the query result set.
 |<<query-set, SET>>                          | Writing     |  Update labels on nodes and properties on nodes and relationships.
 |<<administration-constraints-syntax-list, SHOW [ALL\|UNIQUE\|NODE [PROPERTY] EXIST[ENCE]\|REL[ATIONSHIP] [PROPERTY] EXIST[ENCE]\|[PROPERTY] EXIST[ENCE]\|NODE KEY] CONSTRAINT[S]>> | Schema | List constraints in the database, either all or filtered on type. Also allows `WHERE` and `YIELD` clauses.
-|<<administration-indexes-list-indexes, SHOW [ALL\|BTREE\|FULLTEXT\|LOOKUP] INDEX[ES]>> | Schema | List indexes in the database, either all or filtered on b-tree, fulltext or token lookup indexes. Also allows `WHERE` and `YIELD` clauses.
+|<<administration-indexes-list-indexes, SHOW [ALL\|FULLTEXT\|LOOKUP] INDEX[ES]>> | Schema | List indexes in the database, either all or filtered on b-tree, fulltext or token lookup indexes. Also allows `WHERE` and `YIELD` clauses.
```

Comment:
```suggestion
|<<administration-indexes-list-indexes, SHOW [ALL\|FULLTEXT\|LOOKUP] INDEX[ES]>> | Schema | List indexes in the database, either all or filtered on fulltext, or token lookup indexes. Also allows `WHERE` and `YIELD` clauses.
```

Reply:
```suggestion
|<<administration-indexes-list-indexes, SHOW [ALL\|FULLTEXT\|LOOKUP\|POINT\|RANGE\|TEXT] INDEX[ES]>> | Schema | List indexes in the database, either all or filtered on fulltext, token lookup, point, range, or text indexes. Also allows `WHERE` and `YIELD` clauses.
```

It should probably have the new index types as well though